changelogs/3.3.0.md

```diff
@@ -0,0 +1,206 @@
+  back transactions. When set to 'new', transactions are not rolled back due to
+  a timeout. In this mode `replication.synchro_timeout` is used to wait
+  confirmation in promote/demote and gc-checkpointing. If 'old' is set, the
+  behavior is no different from what it was before this patch appeared.
+* A new `replication.synchro_queue_max_size` option limits the number of
+  transactions in the master synchronous queue.
+  `replication.synchro_queue_max_size` is measured in the number of bytes to be
+  written (0 means unlimited, which was the default behavior before).
+  Currently, this option defaults to 16 megabytes.
+  (gh-7486)
+* Added `box.ctl.replica_gc` function that allows to cleanup
+  resources of a disconnected replica (gh-10755).
+* Now anonymous replicas retain required xlogs. Outdated anonymous replicas
+  and their WAL GC state are deleted automatically when being disconnected
+  for `box.cfg.replication_anon_ttl` seconds (gh-10755).
+
+### Lua
+
+* Introduced a new `offset` parameter in the `index:pairs` method. It allows to
+  skip the first tuples of the iterator (gh-8204).
+* Introduced a new `index:offset_of` method. It allows to get the position in
+  the index relative to the iterator direction of a tuple matching the given
+  key and iterator (gh-8204).
+* Introduced a new `encode_key_order` option for the JSON encoder. It allows
+  you to pass the order of keys in the resulting JSON string (gh-10606).
+* Introduced the `version` Lua module, which enables creation and
+  comparison of version objects. The module eases the use of
+  features blocked before `box.schema.upgrade` (gh-10521).
+
+### Config
+
+* Now users can specify the `lua_call` option to allow calling Lua functions
+  even when the database is in read-only mode or has an outdated schema version
+  (gh-10310).
+* Now, when running in the supervised failover mode, the instance
+  with the highest priority is selected as the bootstrap leader by
+  default (gh-10552).
+* Improved the error message about array data when a record or a map is
+  expected (gh-10241, gh-10242).
+* `config:info()` and its alias `box.info.config` now has the `hierarchy` field
+  that shows the configured instance, replica set, and group names (gh-10308).
+* Added the `isolated` option for instances (gh-10796).
+* Added the `lua.memory` parameter to set up the maximum available RAM for
+  Lua scripts (gh-9849).
+* Added `:jsonschema()` method to the `config` module. This method generates
+  and returns the JSON schema of the cluster configuration, providing a detailed
+  description of each field (gh-10636).
+* Introduced `on_event` callback for roles (gh-10538).
+* Introduced public API for raising alerts (gh-10873).
+* Added the `replication.autoexpel` option (gh-10823).
+
+#### Schema
+
+* Added `<schema object>:jsonschema()` method to
+   `experimental.config.utils.schema` for generating a JSON schema
+   representation as a Lua table (gh-10636).
```

Comment:
This section contains only one bullet, so I would merge it into `### Config`.

Reply:
No problem, done.